frontend/src/app/features/map/siteMarkers/SiteMarker.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC, memo, useState } from 'react';
+import { IconMarker } from '../IconMarker';
+import { LatLngLiteral } from 'leaflet';
+import { mapIconDefault, mapIconHover, mapIconSelected } from './icons';
```

Comment:
Nit but I guess it will be useful: Can we rename it to mapMarkerIcon* since mapIcon is something relevant to map image which is being used in navs..Let me know your thoughts.